src/components/topBar/navigationButtons.css

```diff
@@ -0,0 +1,39 @@
+@import '../app/variablesV2.css';
+
+.wrapper {
+  width: auto;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  margin: 0 15px;
+
+  & > button {
+    width: 15px;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background: transparent;
+    border: none;
+    cursor: pointer;
+    outline: none;
+
+    &:disabled {
+      cursor: default;
+    }
+  }
+
+  & > button:last-child {
+    margin-left: 25px;
+  }
+}
+
+@media (--medium-viewport) {
```

Comment:
We should really hide it in small screens??

Reply:
yes based on the mockups we don't need it in the small views